### PR TITLE
Snap package.

### PR DESCRIPTION
**Pull request by mikeroyal**
Includes a wrapper script and snapcraft.yaml to build [snap](https://snapcraft.io/) for Tensorflow(GPU). I'm still running tests and will make some updates along the way.

### CODE REVIEW

**Comment by rthadur** [comment]
@mikeroyal le t me know when this PR is ready for review. Thank you

File: tensorflow/tools/snapcraft/snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+  wrapper:
+    after:
+      - tensorflow
+    plugin: cmake
+    source: files/
+
+apps:
+ tensorflow:
+    command: bin/wrapper $SNAP/usr/share/tensorflow/bin/tensorflow
```
Comment (perfinion) [inline tensorflow/tools/snapcraft/snap/snapcraft.yaml]: What is this? there is no "tensorflow" binary, its a library. also what does the .desktop file run?

and the url-handler below doesnt make sense either? theres no graphical program to run

Comment (mikeroyal) [reply tensorflow/tools/snapcraft/snap/snapcraft.yaml]: Hi @perfinion, I forgot to remove those components, but I have removed them now

**Comment by mikeroyal** [comment]
Hi @rthadur, I'm ready for review. Let me know if I need to change anything.

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**